### PR TITLE
✨ Add PlaceButton component

### PR DESCRIPTION
**Pull request by MaxMuecket**
- Add PlaceButton.tsx
- Add PlaceButton.stories.tsx
- Add PlaceButton.module.css
https://pocket-garde-place-butt-youlcj.herokuapp.com/storybook/?path=/story/component-placebutton--place

### CODE REVIEW

File: src/app/components/PlaceButton/PlaceButton.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import styles from './PlaceButton.module.css';
+
+const buttonDate = ['sunny', 'half shady', 'shady'];
```
Comment (jujuzitroni) [inline src/app/components/PlaceButton/PlaceButton.tsx:4]:
```suggestion
const buttonPlace = ['sunny', 'half shady', 'shady'];
```

File: src/app/components/PlaceButton/PlaceButton.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  const [currentPlace, setCurrentPlace] = useState(0);
+
+  function handleClick() {
+    setCurrentPlace(currentPlace + 1);
+    if (currentPlace === 2) setCurrentPlace(0);
+  }
+
+  return (
+    <button className={styles.button} onClick={() => handleClick()}>
+      {buttonDate[currentPlace]}
```
Comment (jujuzitroni) [inline src/app/components/PlaceButton/PlaceButton.tsx:16]:
```suggestion
      {buttonPlace[currentPlace]}
```

**Review by jujuzitroni — COMMENTED** [review]
lgtm 👍

**Review by Dev-dfm — APPROVED** [review]
lgtm 🚀

**Review by dhnazarenko — COMMENTED** [review]
LGTM 🚀

**Pull request merged**